lib/generator/models.ex

```diff
@@ -0,0 +1,175 @@
+defmodule Thrift.Generator.Models do
+  def generate(path) do
+    schema = path
+    |> File.read!
+    |> Thrift.Parser.parse
+    |> Map.put(:refs, load_refs(path))
+
+    List.flatten([
+      generate_enums(schema),
+      generate_structs(schema),
+      generate_exceptions(schema),
+    ])
+    |> Enum.map(fn {name, quoted} ->
+      filename = name
+      |> inspect
+      |> String.split(".")
+      |> Enum.map(&Macro.underscore/1)
+      |> Enum.join("/")
+      |> Kernel.<>(".ex")
+      source = Macro.to_string(quoted)
+      {filename, source}
+    end)
+  end
+
+  def namespace(%{namespaces: namespaces}) do
+    case namespaces do
+      %{ex: %{path: path}} ->
+        path |> String.split(".") |> Enum.map(&Macro.camelize/1) |> Enum.join(".")
```

Comment:
The namespace determines both the name of the generated module and the filename in which it is written. By convention Whatever.MyModule goes in whatever/my_module.ex. (http://elixir-lang.org/docs/master/elixir/naming-conventions) This does the conversion from the underscore filename version to the CamelCase module name version. It seems like the namespaces for other languages tend to be defined in the underscore style, so it would be convenient to use that for Elixir also.